x-pack/plugins/fleet/public/applications/fleet/sections/agents/agent_details_page/components/agent_logs/select_log_level.tsx

```diff
@@ -4,30 +4,56 @@
  * you may not use this file except in compliance with the Elastic License.
  */
 
-import React, { memo, useState, useMemo } from 'react';
+import React, { memo, useState, useCallback } from 'react';
 import { i18n } from '@kbn/i18n';
 import { FormattedMessage } from '@kbn/i18n/react';
-import semverGte from 'semver/functions/gte';
 import { EuiSelect, EuiFormLabel, EuiButtonEmpty, EuiFlexItem, EuiFlexGroup } from '@elastic/eui';
 import { Agent } from '../../../../../types';
 import { sendPostAgentAction, useStartServices } from '../../../../../hooks';
+import { AGENT_LOG_LEVELS, DEFAULT_LOG_LEVEL } from './constants';
 
 export const SelectLogLevel: React.FC<{ agent: Agent }> = memo(({ agent }) => {
-  const agentVersion = agent.local_metadata?.elastic?.agent?.version;
-  const isAvailable = useMemo(() => {
-    return semverGte(agentVersion, '7.11.0');
-  }, [agentVersion]);
-
   const { notifications } = useStartServices();
   const [isLoading, setIsLoading] = useState(false);
   const [agentLogLevel, setAgentLogLevel] = useState(
-    agent.local_metadata?.elastic?.agent?.log_level ?? 'info'
+    agent.local_metadata?.elastic?.agent?.log_level ?? DEFAULT_LOG_LEVEL
   );
   const [selectedLogLevel, setSelectedLogLevel] = useState(agentLogLevel);
 
-  if (!isAvailable) {
-    return null;
-  }
+  const onClickApply = useCallback(() => {
+    setIsLoading(true);
+    async function send() {
+      try {
+        const res = await sendPostAgentAction(agent.id, {
+          action: {
+            type: 'SETTINGS',
+            data: {
+              log_level: selectedLogLevel,
+            },
+          },
+        });
+        if (res.error) {
+          throw res.error;
+        }
+        setAgentLogLevel(selectedLogLevel);
+        notifications.toasts.addSuccess(
+          i18n.translate('xpack.fleet.agentLogs.selectLogLevel.successText', {
+            defaultMessage: 'Changed agent logging level to "{logLevel}".',
```

Comment:
I think our copywriting convention would dictate single quotes and no period here
```suggestion
            defaultMessage: `Updated agent logging level to '{logLevel}'`,
```

cc @gchaps

Reply:
All of our other quoted text use single quote I will do the change

Reply:
Yes, the guideline is to use single quotes.